test/test-destruction.js

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) 2017 Intel Corporation. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+const assert = require('assert');
+const rclnodejs = require('../index.js');
+
+describe('Node destroy testing', function() {
+  before(function() {
+    this.timeout(10 * 1000);
```

Comment:
As the performance of the machine used by travis-ci seems not as good as our dev one, the 10s time out is not enough to generate all the necessary js files. I suggest set the time out value to 60s as what I have done on the development branch.